### DISCUSSION TITLE
Add nicer input for game version

### DISCUSSION THREAD

**Discussion by funnyboy-roks** [modrinth/.github#114] [Feature requests]
It would be very nice if there was a better way to input game versions for plugins/mods/resource packs/etc.

I think that one should be able to type `1.13-1.19.4` and it will auto-fill everything from 1.13 until 1.19.4 (inclusive).  This would be very useful for devs who publish releases that work for a lot of different game versions, so we don't have to keep clicking until we get them all.

**Comment by retrooper** [comment]
Is anyone planning on adding this?
